README.md

```diff
@@ -116,16 +116,51 @@ See the [Set up the AWS SDK for Java][docs-setup] section of the developer guide
 
 Once you check out the code from GitHub, you can build it using Maven.
 
+Note: The `archetypes/archetype-lambda` project requires that you have Python 3 in your path as `python`.
```

Comment:
Can you add this info in the archetype-lambda README instead of the main README?
https://github.com/aws/aws-sdk-java-v2/blob/master/archetypes/archetype-lambda/README.md